apps/wallet/src/ui/app/components/accounts/AccountMultiSelect.tsx

```diff
@@ -27,17 +27,15 @@ export function AccountMultiSelect({
 			className="flex flex-col gap-3"
 		>
 			{accounts.map((account) => (
-				<AccountMultiSelectItem
-					key={account.id}
-					account={account}
-					state={
-						account.isLocked
-							? 'disabled'
-							: selectedAccountIDs.includes(account.id)
-							? 'selected'
-							: undefined
-					}
-				/>
+				<ToggleGroup.Item key={account.id} asChild value={account.id}>
+					<div>
```

Comment:
is this extra `div` necessary?

Reply:
@pchrysochoidis yes, because it `ToggleGroup.Item` will inject a ref to it. I have tried to use `forwardRef` but for some reason, it did not work correctly.